tensorflow/core/ops/array_ops.cc

```diff
@@ -1681,8 +1681,8 @@ REGISTER_OP("_MklSlice")
       TF_RETURN_IF_ERROR(c->MakeShapeFromShapeTensor(1, &begin_value));
 
       // NOTE(mrry): We can't use `MakeShapeFromShapeTensor` for `sizes` because
-      // it might contain -1, which can't be represented -1 in the ShapeHandle
-      // would meqan "unknown".
+      // it might contain -1, which can't be represented. (-1 in the ShapeHandle
+      // would mean "unknown".)
       const Tensor* sizes_value = c->input_tensor(3);
```

Comment:
If this is the only difference from the lambda shape function in "Slice", please refactor the function and put it in [common_shape_fns.cc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/common_shape_fns.cc) instead. Please see [_MklConcat](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/array_ops.cc#L3059) for example.